src/node/Launcher.ts

```diff
@@ -627,8 +628,13 @@ function resolveExecutablePath(
     }
   }
   const revisionInfo = browserFetcher.revisionInfo(launcher._preferredRevision);
+
+  const firefoxHelp = `Run "PUPPETEER_PRODUCT=firefox npm install" or "PUPPETEER_PRODUCT=firefox yarn install" to download a supported Firefox browser binary.`;
+  const chromeHelp = `Run npm install or yarn install to download the correct Chromium revision (${launcher._preferredRevision}).`;
```

Comment:
Now that we're here, I'd suggest dropping the `yarn`-specific command. Those who prefer `yarn` know how to use it, so I don't think this adds anything, and removing it simplifies the error message. WDYT?

```suggestion
  const firefoxHelp = `Run \`PUPPETEER_PRODUCT=firefox npm install\` to download a supported Firefox browser binary.`;
  const chromeHelp = `Run \`npm install\` to download the correct Chromium revision (${launcher._preferredRevision}).`;
```